translations/README.bg.md

```diff
@@ -0,0 +1,86 @@
+- [Относно](#about)
+- [Изготвено с](#built-with)
+- [Инсталация](#installation)
+- [Демо](#live-demo)
+
+<br>
+
+### 🌎 *Прочетете на [други езици](./translations/Translations.md)*
+
+<br>
+
+<kbd>[<img title="Deutsch" alt="Deutsch" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/de.svg" width="22">](./translations/README.de.md)</kbd> Скоро
+<kbd>[<img title="Español" alt="Español" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/es.svg" width="22">](./translations/README.es.md)</kbd>
+<kbd>[<img title="Français" alt="Français" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/fr.svg" width="22">](./translations/README.fr.md)</kbd> Скоро
+<kbd>[<img title="Shqip" alt="Shqip" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/br.svg" width="22">](./translations/README.pt_br.md)</kbd>
+<kbd>[<img title="Ukrainian" alt="Ukrainian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ua.svg" width="22">](./translations/README.ua.md)</kbd>
+<kbd>[<img title="Russian" alt="Russian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ru.svg" width="22">](./translations/README.ru.md)</kbd>
+<kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](./translations/README.it.md)</kbd>
+<kbd>[<img title="Čeština" alt="Čeština" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cz.svg" width="22">](./translations/README.cs.md)</kbd>
+<kbd>[<img title="Български" alt="Български" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/bg.svg" width="22">](./translations/README.bg.md)</kbd>
+
+<br>
+
+## 🤔 Относно
+
+Целта на тази репозитория е да генерира `Github Resume` за всеки github потребители.
+
+## 🛠️ Изготвено с
+
+- [React JS](https://reactjs.org/) - Front-End JavaScript библиотека
+- [Material UI](https://material-ui.com/) - React UI Framework
+
+## Инсталация
+
+За да настроите приложението за разработка на вашата локална машина, моля, следвайте инструкциите по-долу:
+
+1. Клонирайте репозиторията на вашата машина:
+
+```bash
+git clone https://github.com/sabesansathananthan/React-GitHub-Resume.git
+cd React-GitHub-Resume
+```
+
+2. Инсталирайте пакетите
+
+    Ако използвате `npm`
+    ```bash
+    npm install
+    ```
+
+    или
+
+    Ако използвате `yarn`
+    ```bash
+    yarn
+    ```
+
+3. Стартирайте сървъра за разработка
+
+    Ако използвате `npm`
+    ```bash
+    npm start
+    ```
+
+    или
+
+    Ако използвате `yarn`
+    ```bash
+    yarn start
+    ```
+
+4. Посетете <http://localhost:3000>
+
+## [Демо на живо](https://react-github-resume.vercel.app/)
+
+[![Deploy с Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 Лиценз
+
+Този проект е лицензиран под лиценза MIT - вижте файла [LICENSE](./LICENSE) за подробности.
```

Comment:
Please, just change the `LICENSE` word to the right language

Reply:
Fixing.
Again, you might want to fix the Russian version.

Reply:
Thank you again for noticing, I'll review that